src/pages/ProfileView.vue

```diff
@@ -1,27 +1,62 @@
 <template>
-  <ul>
-    <li>Name: {{ authStore.user?.name }} {{ authStore.user?.lastName }}</li>
-    <li>Email: {{ authStore.user?.email }}</li>
-  </ul>
-  <router-link to="/" @click="handleLogout">
-    <main-button small>Sair</main-button>
-  </router-link>
-
-  <h2 class="text-neutral-900 text-2xl">Endereços cadastrados</h2>
-
-  <location-card
-    :active="active"
-    :location="location"
-    @set-active="active = !active"
-  />
+  <div>
+    <h3 class="text-4xl mt-16">Perfil</h3>
+    <p class="mt-2">
+      Consulte seus dados, atualize suas informações e endereços.
+    </p>
+  </div>
+
+  <div class="mt-4 bg-red-600 w-auto h-52 flex justify-center items-center">
+    content
+  </div>
+
+  <div class="md:flex md:justify-center md:gap-10">
+    <div>
+      <h3 class="text-2xl mt-10">Dados pessoais</h3>
+      <form class="mt-4">
+        <div class="flex gap-4 mb-4">
+          <text-field label="Nome" name="name" />
+          <text-field label="Sobrenome" name="lastName" />
+        </div>
+
+        <text-field label="Email" name="email" />
+      </form>
+
+      <div class="mt-6">
+        <main-button :disabled="!meta.dirty" class="mr-3"
+          >Atualizar</main-button
+        >
```

Comment:
Aparentemente aqui não ficou bem formatado, e esse botão por ser o submit deveria estar dentro do `form`.

```suggestion
        <main-button :disabled="!meta.dirty" class="mr-3">
          Atualizar
        </main-button>
```

Reply:
Aqui está corrigido (estou adicionando estes comentários para que saibamos o que foi corrigido).